ansible/roles/ocp-workload-aiadv-fuseonline/defaults/main.yml

```diff
@@ -0,0 +1,92 @@
+---
+become_override: false
+ocp_username: opentlc-mgr
+
+ocp_user_needs_quota: True
+
+ocp_user_groups:
+  - OPENTLC-PROJECT-PROVISIONERS
+
+ocp_project: fuse-online-{{guid}}
+#ocp_project: fuse-ignite-{{guid}}
+subdomain_base: $SUBDOMAIN_BASE
+
+quota_requests_cpu: 12
+quota_limits_cpu: 20
+
+quota_requests_memory: '6Gi'
+quota_limits_memory: '20Gi'
+
+quota_configmaps: 10
+quota_pods: 20
+quota_persistentvolumeclaims: 20
+quota_services: 30
+quota_secrets: 30
+quota_requests_storage: 50Gi
+
+build_status_retries: 20
+build_status_delay: 20
+
+deploy_status_retries: 25
+deploy_status_delay: 20
+
+# OCP Limit Range
+pod_min_mem: 10Mi
+pod_max_mem: 6Gi
+container_max_mem: 4Gi
+default_min_mem: 256Mi
+default_max_mem: 1Gi
+default_min_cpu: 250m
+default_max_cpu: 1000m
+
+POSTGRESQL_MEMORY_LIMIT: 512Mi
+PROMETHEUS_MEMORY_LIMIT: 255Mi
+META_MEMORY_LIMIT: 1Gi
+SERVER_MEMORY_LIMIT: 2Gi
+MAX_INTEGRATIONS_PER_USER: 25
+
+# userId / password for AMQ used in syndesisio
+mq_username: amq
+mq_password: topSecret
+
+
+##########          community syndesis          #################
+#SYNDESIS_REGISTRY: docker.io
+#SYNDESIS_REGISTRY_WEB_SUBCONTEXT: syndesis
+#SYNDESIS_TAG: 1.3.10
+#product_name: syndesis
+#ignite_template_name: "{{ product_name }}"
+
+# Corresponds to community syndesis master branch:
+#syndesisio_url: https://raw.githubusercontent.com/syndesisio/syndesis/master
+
+# Corresponds to GPTE fork of community syndesis master branch
+#   This branch implements the following:
+#       1)  DCs in a paused state
+#syndesisio_url: https://raw.githubusercontent.com/gpe-mw-training/syndesis/gpte-{{SYNDESIS_TAG}}
+#syndesisio_sa_yml:  "{{syndesisio_url}}/install/support/serviceaccount-as-oauthclient-restricted.yml"
+#syndesisio_template_yml: "{{syndesisio_url}}/install/syndesis.yml"
+#################################################################
+
+
+
+##########          supported fuse ignite       #################
+SYNDESIS_REGISTRY: registry.access.redhat.com
+SYNDESIS_REGISTRY_WEB_SUBCONTEXT: fuse7
+product_name: fuse-ignite-1.4
```

Comment:
same question here

Reply:
@fridim Fuse Ignite (aka Fuse Online) is installed using this playbook. So the information listed is correct.